.github/workflows/flutter_test.yml

```diff
@@ -81,7 +81,7 @@ jobs:
 
       - name: build apk
         working-directory: ./flutter/example/android
-        run: flutter build apk --debug --target-platform=android-x64
```

Comment:
I believe this was an optimization for CI build times. Any reason to remove this?

Reply:
This was only added a few days ago by myself, because of the incompatibility of the latest flutter beta (`3.27.0-0.1.pre`) and the `androidNativeBundle`. The appending `targe-platform` was only a workaround to keep the ci pipeline working. 
As the maintainer of the plugin fixed this incompatibility, I will revert the changes.
Here is the [issue](https://github.com/getsentry/sentry-dart/issues/2381).

Reply:
I see. On the other hand it **does** reduce build time by about 1.5 minute so I'd vote for keeping only the emulator build on. Your call

Reply:
If it does reduce build time then I guess it's worth keeping